server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -1177,6 +1177,10 @@ public void testScrollWithOngoingSegmentReplication() throws Exception {
     }
 
     public void testPitCreatedOnReplica() throws Exception {
+        assumeFalse(
+            "Skipping the test as its not compatible with segment replication with remote store yet.",
```

Comment:
Could we add the issue number tracking this issue as well? Like how we do for awaitsFix?

Reply:
+1

Reply:
Updated.